paper/break_all_test.cc

```diff
@@ -19,6 +19,7 @@
 #include "ibuckets.h"
 #include "test.h"
 #include "trie.h"
+#include "cstring"
```

Comment:
Shouldn't this be `<cstring>`?

Reply:
I just bought a Xeon Phi workstation, and I recalled (incorrectly!) that your Boggle code was multithreaded. It must have been some code that I wrote that was multithreaded, but it only took a little patching for your code to build successfully on both the Xeon Phi workstation and a more standard Xeon workstation. My apologies, these weren't the best patches, only the simplest that fixed the few build issues.